package.json

```diff
@@ -15,7 +15,7 @@
   "author": "Allan Chau",
   "license": "MIT",
   "dependencies": {
-    "eslint": "^4.16.0",
-    "eslint-plugin-eslint-comments": "^2.0.2"
+    "eslint": "^5.0.1",
```

Comment:
@allanchau, should we put these back to non-operator versions such as `5.0.1`, and use greenkeeper instead?

Reply:
Yeah we can do that, we should also decide if we want to start using Yarn too.
There are some settings to greenkeeper/codefresh that should be made so the lockfile is automatically updated too.
Yarn has also added the ability to read from package-lock.json and update yarn.lock so we could potentially use both across all our projects. https://yarnpkg.com/blog/2018/06/04/yarn-import-package-lock/

Reply:
@allanchau, yep, let's use Yarn. Please make the greenkeeper changes in this branch.

Reply:
In fact, you can't really can you? I'll approve this one, and we can do another PR for that.